mail/rspamd/src/opnsense/service/templates/OPNsense/Rspamd/phishing.conf

```diff
@@ -3,7 +3,9 @@
 #
 {% if helpers.exists('OPNsense.Rspamd.general.enabled') and OPNsense.Rspamd.general.enabled == '1' and helpers.exists('OPNsense.Rspamd.phishing') %}
     openphish_enabled = {% if helpers.exists('OPNsense.Rspamd.phishing.openphish_enabled') and OPNsense.Rspamd.phishing.openphish_enabled == '1' %}true{% else %}false{% endif %};
+    {% if helpers.exists('OPNsense.Rspamd.phishing.openphish_map') and OPNsense.Rspamd.phishing.openphish_map != '' %}openphish_map = "{{ OPNsense.Rspamd.phishing.openphish_map }}";{% endif %}
```

Comment:
you forgot the newline ;)

Reply:
Wow .. too late for PR's .. now the templated config looks good to me